### DISCUSSION TITLE
Grouping release notifications

### DISCUSSION THREAD

**Discussion by OfficialCRUGG** [modrinth/.github#131] [Feature requests]
Many projects have buillds for different loaders and different versions that are all still maintained. So when a mod updates from e.g. 1.1.0 to 1.2.0, that's sometimes 20 or even more versions. I get a notification for each if I follow that project.
Instead, it would make sense to group all releases of a mod within x minutes (or just all releases with the same version number?) into one "<Mod> has released x new versions"

**Comment by triphora** [comment]
Duplicate of #55
